### ISSUE TITLE
Missing required owned element

### ISSUE THREAD

**Issue by gerardo-rodriguez** [cloudfour/pwastats#52]
## Summary

The `menu` role is missing an "owned element".

```
<nav class="u-containProse" role="menu" aria-label="Pagination">
  ...  
</nav>
```

## Details

If you run the Chrome Dev Tools Accessibility Audit, you will get this message for the pagination `<nav>`:

> [Severe] Elements with ARIA roles must ensure required owned elements are present (1)

From "[Required Owned Elements](https://www.w3.org/TR/wai-aria/roles#mustContain)": 
> For example, a `menu` should have at least one instance of a `menuitem`, `menuitemcheckbox`, or `menuitemradio`.

**Comment by gerardo-rodriguez** [comment]
After doing more research, we might benefit from using the `navigation` role instead. 
> If HTML5 is used, the `<nav>` element is supposed to apply the landmark role implicitly. As some browser don’t do this at the time of writing, and to improve the experience for users with older browsers, the navigation role should be added as well.

More info: https://www.w3.org/WAI/tutorials/menus/structure/#identifying-the-menu
